lib/OpenQA/Log.pm

```diff
@@ -116,8 +117,11 @@ sub _log_via_mojo_app ($level, $msg) {
 }
 
 sub _try_logging_to_channel ($level, $msg, $channel) {
-    eval { $channel->$level($msg); };
-    return ($@ ? 0 : 1);
+    try {
+        $channel->$level($msg);
+        return 1
+    }
```

Comment:
```suggestion
    try {
        $channel->$level($msg);
        return 1;
    }
```

```suggestion
    try { $channel->$level($msg); return 1 }
```

Reply:
@Martchus I am not sure I prefer that style. Feel welcome to provide a PR on your own